docs/how-to-guides/connect-wallet-dapp.md

```diff
@@ -0,0 +1,101 @@
+---
+id: connect-wallet-dapp
+---
+
+# How to connect a wallet to a dApp
+
+As a dapp developer, you must integrate a web3 wallet with your application to enable users to interact with your application. Upon integration, you may retrieve account information of the connected user or request to sign & send transactions from the user's account.
+
+## Adena Wallet
```

Comment:
Please include a disclaimer or similar statement indicating that Adena is not the exclusive or official wallet.

Reply:
@moul @zivkovicmilos Can we include a more general disclaimer like "here is a list of available wallets for Gnoland. Note that none of these wallets are official or exclusive, so please use them at your own diligence.", instead of pointing out Adena directly?

Ideally, I'd recommend having "official wallets" in this session later so it's more accessible for devs/users to use.

Reply:
Added a disclaimer:

[3603681](https://github.com/gnolang/gno/pull/1046/commits/36036811fccc3c9a8f8a401acf25088a6434db66)